components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadNotification.kt

```diff
@@ -47,7 +46,8 @@ internal object DownloadNotification {
     @VisibleForTesting
     internal fun createDownloadGroupNotification(
         context: Context,
-        notifications: List<DownloadJobState>
+        notifications: List<DownloadJobState>,
+        notificationAccentColor: Int
```

Comment:
Could we add a tests to verify that the `notificationAccentColor` is getting into the notification? :)

Reply:
Oh, found the way, commit coming :)

Reply:
👍🏽